src/tools/pmempool/rm.c

```diff
@@ -49,8 +49,7 @@
 #include "set.h"
```

Comment:
Because pmempool_rm requires some kind of "interactive" mode. This is not required by pmempool_rm and I don't see any point in making pmempool_rm able to work in interactive mode (e.g. like pmempool_check API).